vm/systemSmartContracts/esdt.go

```diff
@@ -489,6 +494,17 @@ func upgradeProperties(token *ESDTData, args [][]byte, tokenType string) error {
 			token.CanAddSpecialRoles = val
 		case canTransferNFTCreateRole:
 			token.CanTransferNFTCreateRole = val
+		case canCreateMultiShard:
+			if !e.flagNFTCreateONMultiShard.IsSet() {
+				return vm.ErrInvalidArgument
+			}
+			if mintBurnable {
+				return vm.ErrInvalidArgument
+			}
+			if !isCreate {
+				return vm.ErrInvalidArgument
```

Comment:
so we can not create an NFT with the CreateMultiShard optional param activated directly. Right?

Reply:
Yes. It needs to be set on creation